schedule/schedule-appengine/src/main/client/jasify/admin/bookings/admin-bookings-users.html

```diff
@@ -0,0 +1,5 @@
+<a>
+    <span ng-show="match.model.name" bind-html-unsafe="match.model.name | typeaheadHighlight:query"></span>
+    <span ng-show="match.model.email" bind-html-unsafe="match.model.email | typeaheadHighlight:query"></span>
+    <small ng-show="match.model.realName" bind-html-unsafe="match.model.realName | typeaheadHighlight:query"></small>
+</a>
```

Comment:
I start by saying that there is absolutely nothing wrong with creating a file like this one, but... In just in case the author is unaware of having "inline" templates I will give an example that could have been used inside `admin-bookings.html` and would probably work with the typeahead.  Is like having a template inside of a template :-)
~~~html
<script type="text/ng-template" id="admin-bookings-users.html">
  <a>
    <span ng-show="match.model.name" bind-html-unsafe="match.model.name | typeaheadHighlight:query"></span>
  </a>
</script>
~~~

If I'm not mistaken, you can use the value of `id` as the templateUrl for anything